Bump version to 0.7.19 (#14431)
astral-sh/uv: CHANGELOG.md

<!-- prettier-ignore-start -->

## 0.7.19

> The **[uv build backend](https://docs.astral.sh/uv/concepts/build-backend/) is now stabilized**, and considered ready for production use.

Reviewer: ```suggestion
The **[uv build backend](https://docs.astral.sh/uv/concepts/build-backend/) is now stable** and considered ready for production use.
```

> The uv build backend is a great choice for most Python projects that are using uv. It has reasonable defaults, with the goal of requiring zero configuration for most users, but provides flexible configuration that allows most Python project structures. It integrates tightly with uv, to improve messaging and user experience. It validates project metadata and structures, preventing common mistakes. And, finally, it's very fast.

Author: I figure we can post elsewhere about the speed? Otherwise, we could append

 "— `uv sync` on a new project (from `uv init`) is 10-30x faster than other build backends"

Reviewer: I'd add that last point, yeah.

> The uv build backend is a great choice for most Python projects that are using uv. It has reasonable defaults, with the goal of requiring zero configuration for most users, but provides flexible configuration that allows most Python project structures. It integrates tightly with uv, to improve messaging and user experience. It validates project metadata and structures, preventing common mistakes. And, finally, it's very fast.

Reviewer: ```suggestion
The uv build backend is a great choice for pure Python projects. It has reasonable defaults, with the goal of requiring zero configuration for most users, but provides flexible configuration to accommodate most Python project structures. It integrates tightly with uv, to improve messaging and user experience. It validates project metadata and structures, preventing common mistakes. And, finally, it's very fast.
```

Author: "that are using uv" is reasonable to drop, it's good for everyone :)
